[2023-04-13T13:44:08.400+0000] {taskinstance.py:1090} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather_dag.inject_data manual__2023-04-13T13:44:02.435314+00:00 [queued]>
[2023-04-13T13:44:08.413+0000] {taskinstance.py:1090} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather_dag.inject_data manual__2023-04-13T13:44:02.435314+00:00 [queued]>
[2023-04-13T13:44:08.414+0000] {taskinstance.py:1288} INFO - 
--------------------------------------------------------------------------------
[2023-04-13T13:44:08.416+0000] {taskinstance.py:1289} INFO - Starting attempt 1 of 1
[2023-04-13T13:44:08.417+0000] {taskinstance.py:1290} INFO - 
--------------------------------------------------------------------------------
[2023-04-13T13:44:08.435+0000] {taskinstance.py:1309} INFO - Executing <Task(SqliteOperator): inject_data> on 2023-04-13 13:44:02.435314+00:00
[2023-04-13T13:44:08.441+0000] {standard_task_runner.py:55} INFO - Started process 2067 to run task
[2023-04-13T13:44:08.450+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'weather_dag', 'inject_data', 'manual__2023-04-13T13:44:02.435314+00:00', '--job-id', '177', '--raw', '--subdir', 'DAGS_FOLDER/weather_dag.py', '--cfg-path', '/tmp/tmplkddkzpa']
[2023-04-13T13:44:08.452+0000] {standard_task_runner.py:83} INFO - Job 177: Subtask inject_data
[2023-04-13T13:44:08.533+0000] {task_command.py:389} INFO - Running <TaskInstance: weather_dag.inject_data manual__2023-04-13T13:44:02.435314+00:00 [running]> on host ffea11fb791c
[2023-04-13T13:44:08.641+0000] {taskinstance.py:1518} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=weather_dag
AIRFLOW_CTX_TASK_ID=inject_data
AIRFLOW_CTX_EXECUTION_DATE=2023-04-13T13:44:02.435314+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-04-13T13:44:02.435314+00:00
[2023-04-13T13:44:08.643+0000] {sql.py:254} INFO - Executing: 
        INSERT INTO measures_ext (timestamp, temp, humidity, cloudiness, wind_speed) VALUES
        (1681392968,
        287.66,
        45,
        "scattered clouds",
        1.71);
        
[2023-04-13T13:44:08.653+0000] {base.py:73} INFO - Using connection ID 'measure_db' for task execution.
[2023-04-13T13:44:08.840+0000] {base.py:73} INFO - Using connection ID 'measure_db' for task execution.
[2023-04-13T13:44:08.847+0000] {sql.py:375} INFO - Running statement: 
        INSERT INTO measures_ext (timestamp, temp, humidity, cloudiness, wind_speed) VALUES
        (1681392968,
        287.66,
        45,
        "scattered clouds",
        1.71);
        , parameters: None
[2023-04-13T13:44:08.863+0000] {sql.py:384} INFO - Rows affected: 1
[2023-04-13T13:44:08.889+0000] {taskinstance.py:1332} INFO - Marking task as SUCCESS. dag_id=weather_dag, task_id=inject_data, execution_date=20230413T134402, start_date=20230413T134408, end_date=20230413T134408
[2023-04-13T13:44:08.942+0000] {local_task_job.py:212} INFO - Task exited with return code 0
[2023-04-13T13:44:08.961+0000] {taskinstance.py:2596} INFO - 0 downstream tasks scheduled from follow-on schedule check
